java/ql/test/library-tests/frameworks/android/slice/test.ql

```diff
@@ -0,0 +1,9 @@
+import java
+import TestUtilities.InlineFlowTest
+import semmle.code.java.dataflow.FlowSources
+
+class SliceValueFlowConf extends DefaultValueFlowConf {
+  override predicate isSource(DataFlow::Node source) {
+    super.isSource(source) or source instanceof RemoteFlowSource
```

Comment:
Not need right now but we should consider if this should be the default source behaviour (jaxrs does the same)